[Feature issue 128] Select a spaceship

MaferRod opened the pull request:
## 👉 Checklist


- [x] I have reviewed my submission thoroughly.
- [x] I have tested my code (if submission is related to coding) and run the game before pushing (to make sure the project compile).
- [x] I have run the JUnit tests (if submission is related to coding).
- [x] I have read the [Code of Conduct](https://github.com/dylan-power/dinosaur-exploder/blob/main/CODE_OF_CONDUCT.md) and [Contribution Guidelines](https://github.com/dylan-power/dinosaur-exploder/blob/main/CONTRIBUTING.md).


## ✍ Description of the Pull Request

Hi, I am new to the open source world, but this is my contribution to the project. :) 
Pressing the start game button will take you to a lobby where you can select the ship you want to play with, at the moment there are 8 ships available to play with. The ShipSelectionMenu file allows you to dynamically add more ships to play with, this menu stores the ship number you want to play with and nests it with the route.
- All images must have the same name where only the ship number changes: spaceshipX.png, X is the ship number (1,2,3,...)
- All images must have the same size of 87 x 102:
- I use this site to resize the images: https://watermarkly.com/es/photo-resizer/
- And the following site to remove the background of the ships and have them in png format: https://www.remove.bg/es/upload


## 🔗 Issue link


- [x] This Pull Request fixes the issue : 'Closes #128 https://github.com/dylan-power/dinosaur-exploder/issues/128
- [ ] This Pull Request does not fix an issue. 

<br>

MaferRod committed:
Select a spaceship

jvondermarck reviewed (approved):
Hello @MaferRod 

That is INSANE !! I love it, great work for real !!

I am really sorry but a PR before you has been raised, so I accepted it first, so now you have just three tiny conflicts, could you resolve then so that we can merge ? Please, thanks !! (It is a PR related to translation)

I will also create two issues on GitHub (just to make people contribute and always have some work on this project ahah) because when you use the mouse to select the spaceship in the lobby, you have a kind of "shaking" look the video, so maybe this could be fix : 

https://github.com/user-attachments/assets/524fe78b-63df-45b3-bc2c-b258b36281d6

And also it would be great to select the spaceship with the keyboard.

Anyway that's awesome work, thanks for contributing !

MaferRod commented:
Thanks! I Will be working on it in te next hours.:)

MaferRod committed:
Merge branch 'main' into SelectSpaceship

MaferRod committed:
Update ShipSelectionMenu.java

I update the menu, now it only marks the spaceship, without the square, hope you like it.

MaferRod committed:
Update BombComponent.java

Fixing duplicate images

MaferRod committed:
Update BombComponent.java

jvondermarck commented:
Awesome thank you !!! I like the thing you added !
